src/main/java/com/cburch/logisim/std/io/PortHDLGeneratorFactory.java

```diff
@@ -9,27 +9,17 @@
 
 package com.cburch.logisim.std.io;
 
-import com.cburch.logisim.data.AttributeSet;
 import com.cburch.logisim.data.BitWidth;
 import com.cburch.logisim.fpga.designrulecheck.Netlist;
 import com.cburch.logisim.fpga.designrulecheck.NetlistComponent;
 import com.cburch.logisim.fpga.hdlgenerator.AbstractHDLGeneratorFactory;
 import com.cburch.logisim.fpga.hdlgenerator.HDL;
 import com.cburch.logisim.fpga.hdlgenerator.HDLGeneratorFactory;
+
 import java.util.ArrayList;
 
 public class PortHDLGeneratorFactory extends AbstractHDLGeneratorFactory {
 
-  @Override
-  public boolean IsOnlyInlined() {
-    return true;
-  }
-
-  @Override
-  public boolean HDLTargetSupported(AttributeSet attrs) {
-    return true;
-  }
-
   @Override
   public ArrayList<String> GetInlinedCode(Netlist nets, Long componentId, NetlistComponent componentInfo, String circuitName) {
     final var contents = new ArrayList<String>();
```

Comment:
Can you please refactor this code to use `LineBuffer` insteadf of `ArrayList<String>`?

Reply:
See my comment above concerning HDLGeneratorFactory

Reply:
> See my comment above concerning HDLGeneratorFactory

Unfortunatelly that's bit tricky on Github as it hides comments from resolved conversations etc. Can you either paste you reply or give direct link?

Reply:
This I will not yet change in this PR, as HDLGeneratorFactory is "everywhere". Refactoring of HDLGeneratorFactory will be a seperate PR